src/commands/context/delete.ts

```diff
@@ -0,0 +1,77 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+
+import { CheHelper } from '../../api/che'
+import { CheApiClient } from '../../api/che-api-client'
+import { CheServerLoginManager } from '../../api/che-login-manager'
+import { KubeHelper } from '../../api/kube'
+import { cheApiEndpoint, cheNamespace, CHE_API_ENDPOINT_KEY, username, USERNAME_KEY } from '../../common-flags'
+
+export default class Delete extends Command {
+  static description = 'switches to another login session'
```

Comment:
Copy paste probably.

Reply:
Yes, will fix. Thank you for pointing out.